akka-stream-tests/src/test/scala/akka/stream/impl/fusing/GraphInterpreterSpecKit.scala

```diff
@@ -17,27 +17,27 @@ import scala.collection.{ Map ⇒ SMap }
 object GraphInterpreterSpecKit {
 
   /**
-   * Create logics and enumerate stages and ports
+   * Create logics and enumerate operators and ports
    *
-   * @param stages Stages to "materialize" into graph stage logic instances
-   * @param upstreams Upstream boundary logics that are already instances of graph stage logic and should be
-   *                  part of the graph, is placed before the rest of the stages
-   * @param downstreams Downstream boundary logics, is placed after the other stages
-   * @param attributes Optional set of attributes to pass to the stages when creating the logics
+   * @param operators Operators to "materialize" into operator logic instances
+   * @param upstreams Upstream boundary logics that are already instances of operator logic and should be
+   *                  part of the graph, is placed before the rest of the operators
+   * @param downstreams Downstream boundary logics, is placed after the other operators
+   * @param attributes Optional set of attributes to pass to the operators when creating the logics
    * @return Created logics and the maps of all inlets respective outlets to those logics
    */
   private[stream] def createLogics(
-    stages:      Array[GraphStageWithMaterializedValue[_ <: Shape, _]],
+    operators:   Array[GraphStageWithMaterializedValue[_ <: Shape, _]],
```

Comment:
Replacing the parameter name, to be in line with the above Scaladoc comment change.

Reply:
this rename I'd skip; it really is stages here "internally"